src/main/java/io/cdap/e2e/pages/locators/CdfNameSpaceAdminLocators.java

```diff
@@ -0,0 +1,124 @@
+
+  @FindBy(how = How.XPATH, using = "//div[@data-testid='create-namespace-name']")
+  public static WebElement namespaceName;
+
+  @FindBy(how = How.XPATH, using = "//input[@placeholder='Namespace description' and @type='text']")
+  public static WebElement namespaceDescription;
+
+  @FindBy(how = How.XPATH, using = "//button[@data-cy='wizard-next-btn']")
+  public static WebElement clickNext;
+
+  @FindBy(how = How.XPATH, using = "//button[@data-cy='wizard-finish-btn']")
+  public static WebElement clickFinish;
+
+  public static WebElement locateKeyProperty(int row) {
+    String xpath = "//*[@data-cy= 'key-value-pair-" + row + "']//input[@placeholder='key']";
+    return SeleniumDriver.getDriver().findElement(By.xpath(xpath));
+  }
+
+  public static WebElement locateValueProperty(int row) {
+    String xpath = "//*[@data-cy= 'key-value-pair-" + row + "']//input[@placeholder='value']";
+    return SeleniumDriver.getDriver().findElement(By.xpath(xpath));
+  }
+
+  public static WebElement locateAddRowButtonProperty(int row) {
+    String xpath = "//*[@data-cy= 'key-value-pair-" + row + "']//span//button[@type='submit' and " +
+        "@class='btn add-row-btn btn-link']";
+    return SeleniumDriver.getDriver().findElement(By.xpath(xpath));
+  }
+
+  @FindBy(how = How.XPATH, using = "//a[contains(text(),\"Switch to\")]")
+  public static WebElement switchToCreatedNamespace;
+
+  @FindBy(how = How.XPATH, using = "//a[contains(text(),'Go to homepage')]")
+  public static WebElement goToHomepage;
+
+  @FindBy(how = How.XPATH, using = "//*[@data-cy='wizard-result-icon-close-btn']")
+  public static WebElement closeAddNameSpaceWindow;
+
+  @FindBy(how = How.XPATH, using = "//span[@id='setpreferences-fd']")
+  public static WebElement setPreferencesfromMainMenu;
+
+  @FindBy(how = How.XPATH, using = "//button[@data-testid='save-prefs-btn']")
+  public static WebElement clickSaveClose;
+
+  @FindBy(how = How.XPATH, using = "//span[contains(text(),'Upload Driver')]")
+  public static WebElement uploadDrivertab;
+
+  @FindBy(how = How.XPATH, using = "//*[contains(@class,'close')]")
+  public static WebElement closeuploadDriverWindow;
+
+  @FindBy(how = How.XPATH, using = "//span[contains(text(),'Link Repository')]")
+  public static WebElement linkRepositoryTab;
+
+  @FindBy(how = How.XPATH, using = "//button[@class=\"MuiButtonBase-root MuiIconButton-root\"]")
+  public static WebElement closeExternalPage;
+
+  @FindBy(how = How.XPATH, using = "//span[contains(text(),'Add Connection')]")
+  public static WebElement addConnectionTab;
+
+  @FindBy(how = How.XPATH, using = "//span[contains(text(),'Import')]")
+  public static WebElement importConnectionTab;
+
+  @FindBy(how = How.XPATH, using = "//span[contains(text(),'Edit')]")
+  public static WebElement editPreferencesbutton;
+
+  @FindBy(how = How.XPATH, using = "//span[contains(text(),'Create Profile')]")
+  public static WebElement createProfileinNamespaceAdmin;
```

Comment:
Rename it to **createProfileInNamespaceAdmin**

Reply:
Resolved.